micro/test/service_test.go

```diff
@@ -66,11 +66,11 @@ func TestServiceBasics(t *testing.T) {
 		Name:        "CoolAddService",
 		Version:     "0.1.0",
 		Description: "Add things together",
-		Endpoint: micro.Endpoint{
+		Schema:      micro.Schema{Request: "", Response: ""},
```

Comment:
just realized - shouldn't we have schema per endpoint?

Reply:
@Jarema I discussed this with Roland, and the schema would be in a single option. If the schema maps the endpoint name to the operations...

Reply:
@aricart agreed, one thing I am wondering about is whether in that case we should have the schema as:
```go
	Schema struct {
		Request  string `json:"request"`
		Response string `json:"response"`
	}
```
with separation of `Request` and `Response`.

Instead, we can have a single schema string with each endpoint containing 2 entries - one for request and one for response (similarly to how it's done in openAPI).

Reply:
@Jarema @ripienaar had a discussion with @piotrpio and I believe we need to take a stance now on how this is presented for multiple endpoints. @piotrpio is making the ADR changes based on what we discussed